src/prefect/serialization/environment.py

```diff
@@ -82,6 +82,44 @@ class Meta:
     metadata = JSONCompatible(allow_none=True)
 
 
+def _generate_metadata(env):
+    fields = [
+        "job_template_path",
+        "job_template",
+        "env",
+        "memory_limit",
+        "memory_request",
+        "cpu_limit",
+        "cpu_request",
+    ]
+    d = env.metadata.copy() if env.metadata else {}
+    for f in fields:
+        d[f] = getattr(env, f)
+    if env.image or d.get("image") is None:
+        d["image"] = env.image
+    return d
+
+
+class KubernetesJobConfigSchema(ObjectSchema):
+    # XXX: this is all a hack to avoid updating cloud/server for now while
+    # experimenting. None of this is good.
+
+    class Meta:
+        object_class = Environment
+        exclude_fields = ["type"]
+
+    labels = fields.List(fields.String())
+    metadata = fields.Function(_generate_metadata, lambda x: x)
```

Comment:
Question: are you doing this so that all run configs have a universal schema with only 3 fields?  I think that's what we had talked about, but just confirming the reasoning

Reply:
Oops, missed deleting all this. You want to look at `prefect.serialization.run_config`, this was leftover from the hack just for testing without messing with cloud.